app/views/layouts/application.html.erb

```diff
@@ -28,15 +28,14 @@
 
             <div class="pull-right user-info"
               <% if current_user %>
-                Hello <%= link_to (current_user.name || current_user.email), current_user %>!
+                Hello <%= link_to (current_user.name || current_user.email) %>!
```

Comment:
Looks like this link will only display the user's name or email but not actually link to anything. Was that intentional?